bittensor/synapses/dpn/model.py

```diff
@@ -17,22 +17,22 @@ class DPNSynapse(bittensor.Synapse):
     """
 
     def __init__(   self, 
-                    config: DPNConfig,
+                    config: DPNConfig = None,
```

Comment:
should we allow this to be None? it'd break the model init

Reply:
Shouldn't break because we have a check in the init